examples/object/policy/observe.yaml

```diff
@@ -6,14 +6,14 @@ metadata:
 data:
   sample-key: sample-value
 ---
-apiVersion: kubernetes.crossplane.io/v1alpha1
+apiVersion: kubernetes.crossplane.io/v1beta1
 kind: Object
 metadata:
   name: foo
 spec:
-  # Use management policy Observe to observe k8s resource,
+  # Use management policies Observe to observe k8s resource,
   # but leave to third party to create, update, or delete the resource
-  managementPolicy: Observe
+  managementPolicies: ["Observe"]
```

Comment:
I am pretty sure you did, but could you confirm that this manifest working as expected, e.g. if we delete MR, actual k8s object not deleted? Same for other policies as well.